exporter/signalfxexporter/testdata/config.yaml

```diff
@@ -28,6 +28,22 @@ exporters:
     - action: rename_dimension_keys
       mapping:
         k8s.cluster.name: kubernetes_cluster
+    - action: drop_dimensions
+      without_dimension_pairs:
```

Comment:
the `without_` prefix sounds a bit confusing to me. can't we reuse existing `dimension_key` and `dimension_values` options?

Reply:
`without_` sounds like double negation

Reply:
Updated to `dimension_pairs`. I considered using the existing options. However, if someone wants to drop multiple dimension keys, then they would essentially need one rule per dimension key even if the set of metrics to operate on are the same. I think it would be nice to be able to operate over multiple dimension keys in a single rule.

Reply:
Ok, sounds good